src/index.md

```diff
@@ -1370,6 +1370,139 @@ The fix for the main issue should remove the test from the ProblemList or remove
 
 After a failure is handled by excluding a test, the main JBS issue should be re-triaged and possibly given a new priority. This should be handled by the standard triage process. A test exclusion results in an outage in our testing. This outage should be taken into consideration when triaging, in addition to the impact of the bug itself.
 
+## Backing Out a Change
+
+If a change causes a regression that can't be fixed within reasonable time the best way to handle the regression can be to back out the change. Backing out means that the inverse (anti-delta) of the change is pushed to effectively undo the change in the repository. There are two parts to this task, how to do the bookkeeping in JBS, and how to do the actual backout in git or mercurial.
+
+The backout is a regular change and will have to go through the standard code review process, but is considered a [trivial](#trivial) change. The rationale is that a backout is usually urgent in nature and the change itself is automatically generated. In areas where two reviewers are normally required, only one additional Reviewer is required for a backout since the person who is performing the backout also will review the change.
```

Comment:
Recommend that both instances of "reviewer*" in the final sentence be of the same case.  I would choose upper.

Reply:
The first "reviewers" does not refer to two Reviewers. For instance in Hotspot two reviewers are required but only one of them have to be a Reviewer.

Reply:
Ah! Thanks for the clarification.  The existing case make sense.